cmd/snap-confine/snap-confine-invocation.h

```diff
@@ -38,6 +38,7 @@ typedef struct sc_invocation {
     /* Things derived at runtime. */
     char *base_snap_name;
     char *rootfs_dir;
+    char **homedirs;
```

Comment:
Can you also add `num_homedirs` or similar which would then allow to remove the code which counts `homedirs` in `homedirs_are_mounted()` and `sc_homedir_mounts()` etc?

Reply:
Nice suggestion, it simplified the code a few places. Done